src/main/java/net/neoforged/moddevgradle/internal/RunUtils.java

```diff
@@ -230,6 +233,16 @@ private static Project findSourceSetProject(Project someProject, SourceSet sourc
         throw new IllegalArgumentException("Could not find project for source set " + someProject);
     }
 
+    public static Map<String, String> replaceModClassesEnv(RunModel model, Supplier<ModFoldersProvider> newProvider) {
```

Comment:
Not a fan of this. Will we be stuck with the nasty environment variable forever? If that's the case, might as well forget about `--fml.modFolders`.